packages/react-components/src/shared/src/mergeClasses.ts

```diff
@@ -0,0 +1,13 @@
+export function mergeClasses(...values: (string | false)[]) {
```

Comment:
Should you add a return type?

Reply:
Not needed, it is inferred by typescript since .join returns a string